src/workerd/io/compatibility-date.capnp

```diff
@@ -424,12 +424,20 @@ struct CompatibilityFlags @0x8f8c1b68151b6cef {
   pythonWorkers @43 :Bool
       $compatEnableFlag("python_workers")
       $pythonSnapshotRelease(pyodide = "0.26.0a2", pyodideRevision = "2024-03-01",
-          packages = "2024-03-01", backport = 0);
+          packages = "2024-03-01", backport = 0)
+      $impliedByAfterDate(name = "pythonWorkersDevPyodide", date = "2000-01-01");
```

Comment:
Not sure if I'm understanding this bit. If I set my compatibility date to 2024-02-29, does this mean the `python_workers_development` flag will be set automatically?

Reply:
I believe that it says if `pythonWorkersDevPyodide` is on and the compat date is after `2000-01-01` (aka always) then `pythonWorkers` should be on too.